client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -49,20 +96,50 @@ class SpeedUpSiteSettings extends Component {
 				<Card>
 					<FormFieldset className="site-settings__formfieldset">
 						<SupportInfo
-							text={ translate( 'Hosts your image files on the global WordPress.com servers.' ) }
-							link="https://jetpack.com/support/photon/"
-						/>
-						<JetpackModuleToggle
-							siteId={ selectedSiteId }
-							moduleSlug="photon"
-							label={ translate( 'Serve images from our servers' ) }
-							description={ translate(
-								'Jetpack will optimize your images and serve them from the server ' +
-									'location nearest to your visitors. Using our global content delivery ' +
-									'network will boost the loading speed of your site.'
+							text={ translate(
+								"Jetpack's global Content Delivery Network (CDN) optimizes " +
+									'files and images so your visitors enjoy ' +
+									'the fastest experience regardless of device or location.'
 							) }
-							disabled={ isRequestingOrSaving || photonModuleUnavailable }
+							link="http://jetpack.com/support/site-accelerator/"
 						/>
+						<p className="site-settings__feature-description form-setting-explanation">
+							{ translate(
+								'Load pages faster by allowing Jetpack to optimize your images and serve your images ' +
+									'and static files (like CSS and JavaScript) from our global network of servers.'
+							) }
+						</p>
+						<CompactFormToggle
+							checked={ siteAcceleratorStatus }
+							disabled={
+								isRequestingOrSaving ||
+								photonModuleUnavailable ||
+								! siteAcceleratorSupported ||
```

Comment:
I expected that if Site Accelerator is not supported (meaning, we're running an older Jetpack version), we'd see the old UI. However, the PR suggests that we just make the fields inactive/disabled.

I think it could be confusing for the users to just see disabled fields without explanation - perhaps it's best to ask for someone from @Automattic/jetpack-design for opinion on how to best proceed here so it's the most intuitive for the user?